client/my-sites/stats/activity-log-item/index.jsx

```diff
@@ -399,7 +399,9 @@ class ActivityLogItem extends Component {
 				) }
 				<div className={ classes }>
 					<div className="activity-log-item__type">
-						<div className="activity-log-item__time">{ adjustedTime.format( 'LT' ) }</div>
+						<div className="activity-log-item__time" title={ adjustedTime.format( 'LTS' ) }>
+							{ adjustedTime.format( 'LT' ) }
+						</div>
```

Comment:
nice update, but can we take it a step further and use the HTML `<time>` attribute?

```js
<time className="activity-log-item__time" title={ adjustedTime.format() }>{ adjustedTime.format( 'LT' ) }</time>
```